src/components/EditableHeading/__stories__/EditableHeading.stories.mdx

```diff
@@ -0,0 +1,70 @@
+import EditableHeading from "../EditableHeading";
+import { ArgsTable, Canvas, Meta, Story } from "@storybook/addon-docs";
+import { createStoryMetaSettingsDecorator } from "../../../storybook";
+import { createComponentTemplate } from "vibe-storybook-components";
+import { useState } from "react";
+import { overviewPlaySuite } from "../__tests__/EditableHeading-interactions";
+
+export const metaSettings = createStoryMetaSettingsDecorator({
+  component: EditableHeading,
+  enumPropNamesArray: ["weight", "type"],
+  actionPropsArray: ["onChange"]
+});
+
+<Meta
+  title="Inputs/EditableHeading"
+  component={EditableHeading}
+  argTypes={metaSettings.argTypes}
+  decorators={metaSettings.decorators}
+/>
+
+export const editableHeadingTemplate = createComponentTemplate(EditableHeading);
+
+# EditableHeading
+
+- [Overview](#overview)
+- [Props](#props)
+- [Usage](#usage)
+- [Variants](#variants)
+- [Feedback](#feedback)
+
+## Overview
+
+An extension of the <Link href="/?path=/docs/text-heading--overview" withoutSpacing>Heading</Link> component with built in editing capabilities.
+
+<Canvas>
+  <Story
+    name="Overview"
+    args={{
+      value: "This heading is an editable heading",
+      type: EditableHeading.types.H1
+    }}
+    play={overviewPlaySuite}
+  >
+    {editableHeadingTemplate.bind({})}
+  </Story>
+</Canvas>
+
+## Props
+
+<ArgsTable story="Overview" />
+
+## Usage
+
+<UsageGuidelines
+  guidelines={[
+    "Use when the title (Heading) is editable and you want to edit it",
+    "The component takes 100% of width and height of its container"
```

Comment:
is it? 
<img width="1130" alt="image" src="https://github.com/mondaycom/monday-ui-react-core/assets/6093192/b4e347b1-a2a5-4b00-9d2c-840db111a87f">

Reply:
I agree but I asked Meirav specifically about this and she said that even though the header itself is an inline element, the entire component takes the entire block which is true